spark-etl/src/main/scala/geotrellis/spark/etl/EtlConf.scala

```diff
@@ -0,0 +1,38 @@
+package geotrellis.spark.etl
+
+import geotrellis.proj4.CRS
+import org.apache.spark.storage.StorageLevel
+import org.rogach.scallop._
+
+class EtlConf(args: Seq[String]) extends ScallopConf(args){
+  val ingest       = opt[String]("ingest", required = true,
```

Comment:
rename `ingest` here to `input`, and then make sure it's consistent through codebase.